worlds/smw/Locations.py

```diff
@@ -0,0 +1,236 @@
+import typing
+
+from BaseClasses import Location
+from .Names import LocationName
+
+
+class SMWLocation(Location):
+    game: str = "Super Mario World"
+
+    progress_byte: int = 0x000000
+    progress_bit:  int = 0
+
+    def __init__(self, player: int, name: str = '', address: int = None, parent=None, prog_byte: int = None, prog_bit: int = None):
+        super().__init__(player, name, address, parent)
+        self.progress_byte = prog_byte
```

Comment:
Since you're always assigning these values here couldn't you just default assign what you want in your args instead of defaulting them on the class? kind of wasteful

Reply:
I'm not sure what you mean.

Reply:
you're initializing values on lines 10 and 11 and immediately overriding them in the init with either whatever value gets passed or `None`